projects/telemetry/inc/nmea.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#include "status.h"
+
+// # The NMEA message format
+// A given NMEA message starts with $, then contains various comma-seperated values,
+// followed by a *, and then the message checksum (which is a 2-digit hex number)
+//
+// The checksum is calculated by taking the XOR of all characters between
+// (but not including) the '$' and '*' character
+//
+// An example NMEA message is "$GPGLL,2503.6319,N,12136.0099,E,053740.000,A,A*52"
+// In this message, the checksum would be 52
+
+// Computes the checksum for a given NMEA message.
```

Comment:
We usually keep function comments in one block. Right now, it's hard to tell where your description of the message format ends and the function comments start.